<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Nirmala UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Applications</string>
  </property>
  <property name="windowIcon">
   <iconset resource="logo/logo.qrc">
    <normaloff>:/logo/logo_icon.png</normaloff>:/logo/logo_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#Form{
background-color: qconicalgradient(cx:0.114, cy:0.113318, angle:134.3, stop:0 rgba(214, 28, 30, 255), stop:1 rgba(255, 255, 255, 255));}</string>
  </property>
  <widget class="QLabel" name="label_APPLICATIONS">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>981</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(214, 30, 32);</string>
   </property>
   <property name="text">
    <string>APPLICATIONS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SEARCH">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgb(214, 30, 32);
	background-color: rgb(253, 253, 253);
}

QPushButton::hover{
	background-color: rgb(226, 92, 93);
	color: rgb(255, 255, 255);
}

QPushButton::pressed{
	background-color: rgb(216, 37, 39);
}</string>
   </property>
   <property name="text">
    <string>SEARCH</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_RETURN_REFERENCE_MENU">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>60</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgb(214, 30, 32);
	background-color: rgb(253, 253, 253);
}

QPushButton::hover{
	background-color: rgb(226, 92, 93);
	color: rgb(255, 255, 255);
}

QPushButton::pressed{
	background-color: rgb(216, 37, 39);
}</string>
   </property>
   <property name="text">
    <string>RETURN PREFERENCE MENU</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_DEFINED_MENTOR_MEETING">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgb(214, 30, 32);
	background-color: rgb(253, 253, 253);
}

QPushButton::hover{
	background-color: rgb(226, 92, 93);
	color: rgb(255, 255, 255);
}

QPushButton::pressed{
	background-color: rgb(216, 37, 39);
}</string>
   </property>
   <property name="text">
    <string>DEFINED MENTOR MEETING</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ALL_APPLICATIONS">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgb(214, 30, 32);
	background-color: rgb(253, 253, 253);
}

QPushButton::hover{
	background-color: rgb(226, 92, 93);
	color: rgb(255, 255, 255);
}

QPushButton::pressed{
	background-color: rgb(216, 37, 39);
}</string>
   </property>
   <property name="text">
    <string>ALL APPLICATIONS</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>461</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:solid;
border-width:3px;
border-radius:20px;
border-color: rgb(218, 30, 60);
color: rgb(104, 104, 104);
background-color: rgb(255, 255, 255);
border-color: rgb(220, 63, 65);</string>
   </property>
   <property name="placeholderText">
    <string>Enter the text you want to search for here</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_DIFFERENT_REGISTRATION">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>180</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgb(214, 30, 32);
	background-color: rgb(253, 253, 253);
}

QPushButton::hover{
	background-color: rgb(226, 92, 93);
	color: rgb(255, 255, 255);
}

QPushButton::pressed{
	background-color: rgb(216, 37, 39);
}</string>
   </property>
   <property name="text">
    <string>DIFFERENT REGISTRATION</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_UNDEFINED_MENTOR_MEETING">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgb(214, 30, 32);
	background-color: rgb(253, 253, 253);
}

QPushButton::hover{
	background-color: rgb(226, 92, 93);
	color: rgb(255, 255, 255);
}

QPushButton::pressed{
	background-color: rgb(216, 37, 39);
}</string>
   </property>
   <property name="text">
    <string>UNDEFINED MENTOR MEETING</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_DUPLICATE_REGISTRATION">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>240</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgb(214, 30, 32);
	background-color: rgb(253, 253, 253);
}

QPushButton::hover{
	background-color: rgb(226, 92, 93);
	color: rgb(255, 255, 255);
}

QPushButton::pressed{
	background-color: rgb(216, 37, 39);
}</string>
   </property>
   <property name="text">
    <string>DUPLICATE REGISTRATION</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_PREVIOUS_VIT_CHECK">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>180</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgb(214, 30, 32);
	background-color: rgb(253, 253, 253);
}

QPushButton::hover{
	background-color: rgb(226, 92, 93);
	color: rgb(255, 255, 255);
}

QPushButton::pressed{
	background-color: rgb(216, 37, 39);
}</string>
   </property>
   <property name="text">
    <string>PREVIOUS VIT CHECK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_APPLICATION_FILTERING">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>120</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgb(214, 30, 32);
	background-color: rgb(253, 253, 253);
}

QPushButton::hover{
	background-color: rgb(226, 92, 93);
	color: rgb(255, 255, 255);
}

QPushButton::pressed{
	background-color: rgb(216, 37, 39);
}</string>
   </property>
   <property name="text">
    <string>APPLICATION FILTERING</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>861</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>136</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>DATE</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>FULL NAME</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>E-MAIL</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>PHONE NUMBER</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>POSTAL CODE</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>PROVINCE</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string> CURRENT STATUS</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources>
  <include location="logo/logo.qrc"/>
 </resources>
 <connections/>
</ui>
